.circleci/config/commands/go_test.yml

```diff
@@ -21,8 +21,9 @@ parameters:
   arch:
     type: string
     # Only supported for use_docker=false, and only other value allowed is 386
-    default: amd64
+    default: amd64 # must be 386 or amd64
 steps:
+  - configure-git
```

Comment:
I assume this will simply do nothing useful on OSS?

Reply:
I mean with respect to the add_ssh_keys

Reply:
Right, it's not _needed_ for OSS, but having these files be the same seems easier to manage for now..

Reply:
And yes, the referenced SSH key doesn't exist on OSS, so that part is just ignored.